### PR TITLE
Update plugin documentation to reference the CAT API

### LINKED ISSUE #1015: [DOC] Add more info about plugins
**What do you want to do?**
 
- [ ] Request a change to existing documentation

**Tell us about your request.** Provide a summary of the request and all versions that are affected.
 
I recommend that the "plugin installation" page (https://opensearch.org/docs/latest/opensearch/install/plugins/) cross-reference the API call "cat plugins" (https://opensearch.org/docs/latest/opensearch/rest-api/cat/cat-plugins/)

Is there any way to list the *available* (not installed) plugins that can be installed by name either on the command line or in an API call?

**What other resources are available?** Provide links to related issues, POCs, steps for testing, etc.

### PR DESCRIPTION
**Pull request by ariamarble**
Signed-off-by: ariamarble <armarble@amazon.com>

### Description

Update documentation

### Issues Resolved

#1015 

### Checklist
- [X] By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license and subject to the [Developers Certificate of Origin](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: _opensearch/install/plugins.md [hunk 1]
```diff
@@ -52,6 +52,29 @@ opensearch-security
 opensearch-sql
 ```
 
+You can also list plugins that are installed by using the [CAT API]({{site.url}}{{site.baseurl}}/api-reference/cat/cat-plugins/):
+
+#### Path and HTTP method
```
Comment (cwillum) [inline _opensearch/install/plugins.md:57]: I think this should be a level-3 heading.

**Review by carolxob — APPROVED** [review]

File: _opensearch/install/plugins.md [hunk 2]
```diff
@@ -52,6 +52,29 @@ opensearch-security
 opensearch-sql
 ```
 
+You can also list plugins that are installed by using the [CAT API]({{site.url}}{{site.baseurl}}/api-reference/cat/cat-plugins/):
```
Comment (natebower) [inline _opensearch/install/plugins.md:55]:
```suggestion
You can also list installed plugins by using the [CAT API]({{site.url}}{{site.baseurl}}/api-reference/cat/cat-plugins/).
```

File: _opensearch/install/plugins.md [hunk 3]
```diff
@@ -52,6 +52,29 @@ opensearch-security
...
+
+#### Path and HTTP method
+
+```bash
+GET _cat/plugins
+```
+
+#### Example response
```
Comment (natebower) [inline _opensearch/install/plugins.md:63]: Ensure consistency across docs (I think we use "Sample" in other docs).

File: _opensearch/install/plugins.md [hunk 4]
```diff
@@ -52,6 +52,29 @@ opensearch-security
...
+opensearch-node1 opensearch-job-scheduler             2.0.1.0
+opensearch-node1 opensearch-knn                       2.0.1.0
+opensearch-node1 opensearch-ml                        2.0.1.0
+opensearch-node1 opensearch-notifications             2.0.1.0
+opensearch-node1 opensearch-notifications-core        2.0.1.0
+```
+
 ## Install
 
 There are three ways to install plugins using `opensearch-plugin`:
```
Comment (natebower) [inline _opensearch/install/plugins.md:80]:
```suggestion
There are three ways to install plugins using the `opensearch-plugin`:
```

**Review by natebower — COMMENTED** [review]
@ariamarble Just a few changes. Thanks!

File: _opensearch/install/plugins.md (hunk 1 above)
Comment (ariamarble) [reply _opensearch/install/plugins.md:57]: for our style guide we actually keep these as L4s

**Pull request merged**